src/SamplesApp/UITests.Shared/Windows_UI_Xaml/MarkupExtensionTests/Simple.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml.Markup;
+
+namespace UITests.Shared.Windows_UI_Xaml.MarkupExtension
+{
+	// Not yet supported in Uno
+#if NETFX_CORE
```

Comment:
It doesn't compile? even if it's not implemented it should compile without #if NETFX_CORE

Reply:
No, it doesn't compile at all on non-UWP platforms.

Reply:
What error does the compiler give?

Reply:
everything should compile (in theory), if this doesn't compile we might want to look at it in higher priority! I reopened this conversation.

Reply:
I will be creating another PR to fix all the merge issues